bdtopo_v2/src/test/groovy/org/orbisgis/orbisprocess/geoclimate/bdtopo_v2/InputDataFormattingTest.groovy

```diff
@@ -113,13 +114,15 @@ class InputDataFormattingTest {
         assertEquals('INTEGER', table.columnType('ZINDEX'))
         assertEquals('VARCHAR', table.columnType('ID_ZONE'))
         // For each rows, check if the fields contains the expected values
-        table.eachRow { row ->
+        println tableName
+        h2GISDatabase.eachRow("SELECT * FROM $tableName") { row ->
             assertNotNull(row.THE_GEOM)
             assertNotEquals('', row.THE_GEOM)
             assertNotNull(row.ID_BUILD)
             assertNotEquals('', row.ID_BUILD)
             assertNotNull(row.ID_SOURCE)
             assertNotEquals('', row.ID_SOURCE)
+            println row.ID_BUILD
             // Check that the HEIGHT_WALL is smaller than 1000m high
             assertNotNull(row.HEIGHT_WALL)
             assertNotEquals('', row.HEIGHT_WALL)
```

Comment:
Here is the problem. And strange thing, I display the ID of each building in order to know where happen the problem. It happens right after id 2231... which is the end of the file...

Reply:
Ok @SPalominos 
Perhaps a pb with last change in OrbisData with the next method

Reply:
Note that it does not happen if I do the same iteration on an input table such as bati_indifferencie

Reply:
@j3r3m1 @ebocher Does this issue still happen ? I can't reproduce it, the test reach the end of the iteration.

Reply:
Yes I still have the problem